landing_page_app/templates/components/base.html

```diff
@@ -22,31 +24,20 @@
 {% endblock head %}
 
 {% block header %}
-    <div class="app-pane__header toc-open-disabled">
-        <header class="govuk-header app-header" role="banner" data-module="govuk-header">
-            <div class="govuk-header__container govuk-header__container--full-width">
-                <div class="govuk-header__logo">
-                    <a class="govuk-header__link govuk-header__link--homepage" href="/">
-          <span class="govuk-header__product-name">
-            Operations-Engineering Landing Page PoC
-          </span>
-                    </a>
-                </div>
-                <div class="govuk-header__content">
-                    <nav class="govuk-header__navigation govuk-header__navigation--end">
-                        <ul class="govuk-header__navigation-list">
-                            <li class="govuk-header__navigation-item">
-                                <a class="govuk-header__link"
-                                   href="https://github.com/ministryofjustice/operations-engineering-landing-page-poc">
-                                    GitHub
-                                </a>
-                            </li>
-                        </ul>
-                    </nav>
-                </div>
-            </div>
-        </header>
-    </div>
+  {{ govukHeader({
+    'homepageUrl': '/',
+    'serviceName': 'Operations-Engineering Landing Page PoC',
+    'serviceUrl': '/'
+  }) }}
+{% endblock %}
+
+{% block beforeContent %}
+  {{ govukPhaseBanner({
+    'tag': {
+      'text': 'DEV'
```

Comment:
Probably a separate ticket, but It'd be good to be able to configure this per environment 🔧 I.e. have `Private Beta` in production, but keep `DEV` in the development environment 👨‍💻

Reply:
Yeah I can create a separate ticket for that